exonum-java-binding/core/src/test/java/com/exonum/binding/core/blockchain/BlockchainDataIntegrationTest.java

```diff
@@ -94,6 +95,31 @@ void getExecutingServiceDataFromFork() throws CloseFailuresException {
     assertThat(entry2.get()).isEqualTo("V1");
   }
 
+  @Test
+  void createIndexProofFromForkDisallowed() throws CloseFailuresException {
+    // Setup the service data, so that the index is initialized
+    String serviceName = "test-service";
+    String simpleIndexName = "test-list";
+    try (Cleaner cl = new Cleaner()) {
+      Fork fork = db.createFork(cl);
+
+      // Use the full index name
+      String fullIndexName = serviceName + "." + simpleIndexName;
```

Comment:
I found that this code is frequently used. Maybe it's time to consider creating a function somewhere?

Reply:
In tests I think it's OK to use it, but where else? The service schemas has been migrated to simple names (they no longer need explicit namespacing since #1393 ).

It might make sense to make it public in the BlockchainData if it is needed in proof creation (but I am not sure it will be needed).